classes/requests/helpers/class-nets-easy-checkout-helper.php

```diff
@@ -70,6 +70,8 @@ public static function get_checkout( $checkout_flow = 'inline', $order_id = null
 			$checkout['consumer']                                = self::get_consumer_address( $order );
 		}
 
+		$checkout['countryCode'] = dibs_get_iso_3_country( isset( $order ) ? $order->get_billing_country() : WC()->customer->get_billing_country() );
```

Comment:
`isset( $order )` will return true if the order is false here. And since `wc_get_order` can return false, this could cause an error. I would recommend moving this into the if/else blocks instead, and handle each case separately  instead, one for the cart and one for the order.